Cargo.toml

```diff
@@ -1,23 +1,37 @@
 [workspace]
-members = ["ceramic-kubo-rpc", "ceramic-one"]
+members = [
+    "core",
+    "event",
+    "ceramic-kubo-rpc",
+    "ceramic-one"
+]
 
 [workspace.dependencies]
 anyhow = "1"
 async-stream = "0.3"
 async-trait = "0.1"
 ceramic-one = { path = "./ceramic-one" }
 dag-jose = "0.1"
+cid = { version = "0.10.1", features = ["serde-codec"] }
+env_logger = "0.10"
```

Comment:
I would not expect to need to use `env_logger` the code base uses `tracing` for its logging needs and the tracing ecosystem has an env filter based subscriber. Did you find this was necessary?

Reply:
It's useful for testing, but I don't think there's a concept of `workspace.dev-dependencies`.

Reply:
I'd expect this is still not used. The tracing crate we use already honors RUST_LOG without this dep.